### PR TITLE
Fix platform name for Openstack clouds

### PR DESCRIPTION
**Pull request by aswinsuryan**
Fix platform name for Openstack clouds

Signed-off-by: Aswin Suryanarayanan <aswinsuryan@gmail.com>

### CODE REVIEW

**Review by vthapar — APPROVED** [review]

File: pkg/cloud/cloud.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func (f *providerFactory) Get(managedClusterInfo configv1alpha1.ManagedClusterIn
 		return gcp.NewGCPProvider(f.restMapper, f.kubeClient, f.dynamicClient, f.hubKubeClient, eventsRecorder,
 			region, infraID, clusterName, config.Spec.CredentialsSecret.Name,
 			config.Spec.GatewayConfig.GCP.InstanceType, config.Spec.IPSecNATTPort, config.Spec.NATTDiscoveryPort, config.Spec.Gateways)
-	case "Openstack":
+	case "OpenStack":
```
Comment (sridhargaddam) [inline pkg/cloud/cloud.go:80]: This looks like a fundamental bug. Just curious, is this the first attempt where we are validating the end-to-end flow with the RHOS PRs?

**Comment by sridhargaddam** [comment]
/lgtm

Can we please ensure that we do end-to-end testing of this feature on RHOS with all the associated PRs and identify any missing elements. Since we do not have a CI job to validate this its important that we test this properly during manual validation.

File: pkg/cloud/cloud.go (hunk 1 above)
Comment (aswinsuryan) [reply pkg/cloud/cloud.go:80]: This was a typo that occurred in the original PR. As I was testing the changes with my local changes it had the correct platform name, but the PR did to have it and was missed in the review.

**Pull request merged**